### PR TITLE
6 get/api/articles/article id/comments

### PR DESCRIPTION
**Pull request by Rachelllk97**
add functionality to retrieve comments by article_id, test accordingly and document in endpoints.json

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -290,3 +286,68 @@ test("should return the array of objects in descing order", () => {
 
 })
 })})})
+
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("should an array of comments for the given article_id and contain correct properties", () => {
```
Comment (xViolaine) [inline __tests__/app.test.js:292]: It should have another test to check if we can get a 200 when the ID is valid, but it has no comments (should return an empty array)

**Review by xViolaine — COMMENTED** [review]
One test is missing here, please add and resubmit! 😄
